### PR TITLE
Integration tests & LMDB map size updates

### PR DESCRIPTION
**Pull request by woky**

### CODE REVIEW

**Comment by woky** [comment]
bors try

**Comment by woky** [comment]
bors try-

**Comment by woky** [comment]
bors try

File: integration-tests/run_tests [hunk 1]
```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```
Comment (marcin-rzeznicki) [inline integration-tests/run_tests:1]: Anything bash specific in here?

**Review by marcin-rzeznicki — APPROVED** [review]

File: integration-tests/run_tests (hunk 1 above)
Comment (woky) [reply integration-tests/run_tests:1]: Well, there's `[[` and `pipefail`. Bash is widespread and less painful than POSIX shell.

Comment (woky) [reply integration-tests/run_tests:1]: FYI this change is just to make it straightforward for devs to run integration tests, not to be too much universal. It used to depend on Bash previously too.

EDIT: And I know I suck for not trying to run it myself before making this PR. On it.

Comment (marcin-rzeznicki) [reply integration-tests/run_tests:1]: > Bash is widespread and less painful than POSIX shell.

... and slow. But, anyway, you're right - ifs are non-POSIXy etc. Got ya, approved :-)

**Comment by woky** [comment]
bors try

**Comment by woky** [comment]
bors try

**Comment by woky** [comment]
bors try

**Review by zsluedem — APPROVED** [review]

**Comment by woky** [comment]
bors try

**Pull request merged**